projects/amd-loader/webpack.config.js

```diff
@@ -36,12 +36,11 @@ module.exports = ({flavor}) => {
 		},
 		min: {
 			mode: 'production',
+			optimization: {
+				minimize: true,
+				minimizer: [new TerserPlugin()],
+			},
```

Comment:
We use `terser` instead of `uglify` because, given that it is used for minification, it should be equivalent. There's however a way to [configure the terser plugin to use uglify](https://webpack.js.org/plugins/terser-webpack-plugin/#custom-minify-function), just in case it is needed.

Reply:
FWIW, uglify [is dead](https://github.com/mishoo/UglifyJS/issues/3156#issuecomment-392943058) (we'll be [using Terser for our own build-time minification](https://github.com/liferay/liferay-frontend-projects/pull/233)).